ddtrace/contrib/django/patch.py

```diff
@@ -408,13 +405,15 @@ def traced_get_response(django, pin, func, instance, args, kwargs):
             if response:
                 status = response.status_code
                 span._set_str_tag("django.response.class", func_name(response))
-                if hasattr(response, "template_name"):
+                try:
```

Comment:
is `try/except` that much faster than `hasattr()`?

Or is it preferred for another reason?

Reply:
As this turns out to be on a hot path, I'd like to squeeze as much as possible, especially if it's a simple change 🙂 .